packages/blocks/drawing-canvas/src/base.css

```diff
@@ -0,0 +1,94 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box
+}
```

Comment:
It'd be nice to scope the CSS if possible. Otherwise, we'll end up with style leaks like here:
https://blockprotocol.org/@kickstartds/blocks/button

Reply:
Agree. Removed the style here => https://github.com/hashintel/hash/pull/515/commits/229dfe48559f79788dac315167cd7ad95af46289.